src/main/java/com/google/error_analyzer/backend/index_template.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Where are we using this file?

Reply:
> Where are we using this file?

We will need to set this index template in elasticsearch